pkg/interface/src/views/apps/links/components/LinkDetail.tsx

```diff
@@ -27,12 +27,13 @@ export function LinkDetail(props: LinkDetailProps) {
   return (
     /*  @ts-ignore indio props?? */
     <Row height="100%" width="100%" flexDirection={['column', 'column', 'row']} {...rest}>
-      <LinkBlockItem minWidth="0" minHeight="0" height={['50%', '50%', '100%']} width={['100%', '100%', 'calc(100% - 350px)']} flexGrow={0} border={0} node={node} />
+      <Center flexBasis={['100%', '100%', '75%']} flexShrink={1} overflowY="scroll" >
+        <LinkBlockItem maxHeight="100%" node={node} />
+      </Center>
       <Col
-        minHeight="0"
-        flexShrink={1}
-        width={['100%', '100%', '350px']}
-        flexGrow={0}
+        flexBasis={['25%', '25%', '350px']}
+        flexShrink={0}
+        flexGrow={1}
```

Comment:
This seems to grow too much with enough content:

![image](https://user-images.githubusercontent.com/5466421/163474287-2ad4341e-b6ed-4119-8d8d-611af553b29e.png)

Reply:
Same issue as mentioned at line 30, see discussion there